src/k8s/K8sInterface.cpp

```diff
@@ -173,3 +173,79 @@ v1_node_list_t *K8sInterface::getNodes() {
                                    NULL);                 /* watch */
     return node_list;
 }
+
+std::string K8sInterface::loadFromConfig(std::string key) {
+    v1_config_map_t *config_map =
+        CoreV1API_readNamespacedConfigMap(apiClient, strdup("jasminegraph-config"), namespace_, NULL);
+    if (config_map->metadata->name == NULL) {
+        k8s_logger.error("No jasminegraph-config config map.");
```

Comment:
Need more context here? Where should this file be pecifically?

Reply:
@Ishad-M-I-M is it the `k8s/configs.yml` ?

Reply:
Yes. `k8s/configs.yml` is the file used to create the config map object.
But this error message is to complain that the config map object is not created

Reply:
Ok